docs/reference-guides/block-api/block-metadata.md

```diff
@@ -602,6 +603,22 @@ It's possible to pass a style handle registered with the [`wp_register_style`](h
 
 _Note: An option to pass also an array of styles exists since WordPress `5.9.0`._
 
+### View Style
+
+-   Type: `WPDefinedAsset`|`WPDefinedAsset[]` ([learn more](#wpdefinedasset))
+-   Optional
+-   Localized: No
+-   Property: `viewStyle`
+-   Since: `WordPress 6.x.x`
```

Comment:
It looks like you can put here and in other places 6.5.0 as all the code included in both Gutenberg and WordPress core seems good enough to move forward with it. I will have a closer look in the following days. Thank you so much for working on the proposal.